### PR TITLE
README: Describe macOS compilation basics more

### PR DESCRIPTION
**Pull request by cjcjameson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,6 +140,12 @@ make cluster
 source gpdemo-env.sh
 ```
 
+If compiling on macOS, make can be parallelized
+
+```
+make -j8
```
Comment (ashwinstar) [inline README.md]: Why only for macOS ? -j8 can be used in general. Seems more relevant to mention at line 130.

Comment (cjcjameson) [inline README.md]: I might be misinformed -- yea it looks like it's across GNU make. Do other makes have it as well.

Comment (danielgustafsson) [inline README.md]: Yes, `-j X` is available in bsd make as well which in turn comes from the old pmake that did parallel builds by default.

Comment (cjcjameson) [inline README.md]: 👍 updated the text. All good?

**Pull request merged**